q2_demux/plugin_setup.py

```diff
@@ -132,7 +136,9 @@
                              'reverse complemented prior to demultiplexing.',
         'rev_comp_mapping_barcodes': 'If provided, the barcode sequences in '
                                      'the sample metadata will be reverse '
-                                     'complemented prior to demultiplexing.'
+                                     'complemented prior to demultiplexing.',
+        'ignore_description_mismatch': 'If True, ignore mismatches in '
```

Comment:
The following edit suggestion will allow this to be interface-agnostic:

```suggestion
        'ignore_description_mismatch': 'If enabled, ignore mismatches in '
```